imessage/bluebubbles/api.go

```diff
@@ -261,7 +261,7 @@ func (bb *blueBubbles) GetMessage(guid string) (resp *imessage.Message, err erro
 	return nil, ErrNotImplemented
 }
 
-func (bb *blueBubbles) apiUrl(path string) string {
+func (bb *blueBubbles) apiUrl(path string, queryParams *map[string]string) string {
```

Comment:
This doesn't need to be a pointer, empty map has the same effect and maps can also be nil

Reply:
i'm so used to Rust haha thank you!